### ISSUE TITLE
[BUG] Docker container is unhealthy with basic auth

### ISSUE THREAD

**Issue by almottier** [benbusby/whoogle-search#316]
**Describe the bug**
Docker container is unhealthy when I use basic auth with env var `WHOOGLE_USER` and `WHOOGLE_PASS`.

**To Reproduce**
```
root@da05c4b66963:/whoogle# wget http://localhost:5000
--2021-05-14 22:41:40--  http://localhost:5000/
Resolving localhost (localhost)... 127.0.0.1, ::1
Connecting to localhost (localhost)|127.0.0.1|:5000... connected.
HTTP request sent, awaiting response... 401 UNAUTHORIZED

Username/Password Authentication Failed.
root@da05c4b66963:/whoogle# echo $?
6
```

**Deployment Method**
- Docker

**Version of Whoogle Search**
- 0.4.1

**Comment by nandor-magyar** [comment]
The app itself includes some of the responsibilites would be better handled by a separate web-server (nginx or else) imo.
I tried to fix this locally, with using a new route that is just returning "" - the `@auth_required` decorator is left out.  
This naturally works and I am happy to send a PR, but I haven't found any formatter/linter guidance in the readme or just missed.

The Dockerfile:
```
HEALTHCHECK  --interval=30s --timeout=5s \
  CMD wget -qO- --no-verbose --tries=1 http://localhost:${EXPOSE_PORT}/healthz || exit 1
```

The new route:
```
@app.route("/healthz", methods=["GET"])
def healthcheck():
    return ""
```

**Comment by benbusby** [comment]
Thanks @nandor-magyar, I just implemented your solution with 1fdf226802d67445f7570b34a3323463e165770d (as well as switching to a `curl` based healthcheck to avoid some problems that another user encountered with `wget`).
